src/v2/features/vs/damage-calculator-service.tsx

```
@@ -0,0 +1,357 @@
+        attacker = this.modifyAttacker(attacker, attacker.relevantTraits, defender.relevantTraits);
+        defender = this.modifyDefender(defender, attacker.relevantTraits, defender.relevantTraits);
+        const ret: number[] = [];
+        const damage: number = attacker.damage;
+        const critDamage: number = attacker.critDamage ?? 0;
+        const blockDamage: number = defender.blockDamage ?? 0;
+        // Modifiers happen after the second pass through gravis, they happen before
+        // blocks, and they are not subject to the usual 20% variance.
+        const minDamage = Math.max(
+            1,
+            hits *
+                this.getDamageFrom1Hit(
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)